tests/Unit/Digitick/Sepa/TransferInformation/CustomerDirectDebitTransferInformationTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+namespace tests\Unit\Digitick\Sepa\TransferInformation;
+
+/***************************************************************
+ *  (c) 2015 networkteam GmbH - all rights reserved
```

Comment:
Ah, damn not a good Moment to create PRs being at symfonyCon. I'll fix it tomorrow